### ISSUE TITLE
Unable to disable "Break on CPU crash"

### ISSUE THREAD

**Issue by jthecodebuilder** [SourMesen/Mesen#649]
I'm messing around in the memory editor, and everytime I freeze and/or unfreeze something critical, the debugger kicks in and halts the emulated CPU execution. I want to be able to disable this (aka make it emulate closer to actual hardware to the point where anything that was still on in the APU would be heard), but it seems that every invalid opcode triggered due to the game crash causes a break to happen, and I am unable to figure out how to actually disable it. I can understand if this is a hardcoded thing to prevent any application/OS-wide crashes, but if so, atleast point to where I can change the code to disable it. Im fine if my OS crashes.

**Comment by jthecodebuilder** [comment]
And yes, I've already tried the actual "Break on CPU crash" option.

**Comment by SourMesen** [comment]
Hey, sorry for the long delay!

If I remember correctly, you should be able to do this by enabling "Developer mode" in Options->Preferences (first tab).  When dev mode is enabled, the CPU crashes will no longer stop/unload the game - all it will do is display a warning message once, and keep executing.

Hope that helps!

**Comment by jthecodebuilder** [comment]
Ok, got around to trying this again, so I enabled the developer mode, and crashed the game using the memory editor (in smb3, freezing and then unfreezing addresses $0069-$006E is an interesting way to do it), but the emulation is still **break**'d, and the debugger pops up. I'm probably missing something, or there is no end to this debugger-enforcement madness in sight (thus likely being a bug). Also, I've noticed disabling the other break option (Break on power/reset) does not work either. Actually, one thing I've noticed, is that what you said only works if you crash the game without using any debug tools. But since I am intentionally crashing the game by messing with its memory via it's editor (which is a debug tool), theoretically, all rules that are applied when the dev mode is enabled to the break commands when no debug tools are enabled are completely overrided, even if the break options are already set to false in the debugger/settings. So it has to be a bug. Another thing I've noticed is that if the emulator is "Power Off"'d after the emulator has break'd due to it detecting the cpu crashed because of the break-enforcing bug, and then resume the emulation by clicking on the state thingy of the respective ROM when nothing is open, the emulation continues without breaking. It also works if you close the Memory Editor window. Basically, what I'm trying to say is **Developer mode disabling these CPU crash breaks only works if you don't open any debug tools, and/or cause the crash within the presence of an open debug tool(s). And even if you do have the breaks disable, they won't work anyway since the darn things are being overided.** And sorry for a looong comment.
